projects/control-service/projects/pipelines_control_service/src/main/resources/k8s-data-job-template.yaml

```diff
@@ -37,4 +37,11 @@ spec:
             runAsUser: 1000
             runAsGroup: 1000
             fsGroup: 1000
+            readOnlyRootFilesystem: true
+          volumeMounts:
```

Comment:
This change requires thought behind UX if we are default to read only. 

Should users be able to create files in the local file system. 

Would this work
```
def run(job_input): 
   with open("my-file", "rw"): 
         # write something   smething ? 
```

And how would it work ?

Reply:
Would this not create the file in the ephemeral volume?

Reply:
I don't know :) I am asking you that actually and hoping you will test/verify and tell us. 

I think if we set those as default configuration which I think make sense we need to make sure some common use-cases are working or if not there's clear (and easy!) way for users to migrate

Reply:
please provide functional test of the local ephemeral storage? Are there any patterns/restrictions on the file paths for end users, or is it completely transparent?